lib/improver/tests/acceptance/test_standardise.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Similar lines in 8 files](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4638888)